!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      state: true
      tf_pitch: 0.0
      tf_roll: 0.0
      tf_x: 1.0
      tf_y: 0.0
      tf_yaw: 0.0
      tf_z: 1.0
      type: ''
    state: []
  tf_pitch: 0.0
  tf_roll: 0.0
  tf_x: 1.0
  tf_y: 0.0
  tf_yaw: 0.0
  tf_z: 1.0
state: []
